hathor/p2p/manager.py

```diff
@@ -225,8 +225,11 @@ def reconnect_to_all(self) -> None:
         """ It is called by the `lc_reconnect` timer and tries to connect to all known
         peers.
 
-        TODO(epnichols): Should we always conect to *all*? Should there be a max #?
+        TODO(epnichols): Should we always connect to *all*? Should there be a max #?
         """
+        if len(self.connected_peers) < 1:
```

Comment:
I think it is a good behavior to add. Please, add some comments explaining what it is doing and why.

Reply:
Added. What do you think?